media-libs/ptex/ptex-2.1.28-r1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Per-Face Texture Mapping for Production Rendering"
+HOMEPAGE="http://ptex.us/"
+SRC_URI="https://github.com/wdas/ptex/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+RDEPEND="sys-libs/zlib"
+DEPEND="${RDEPEND}
+	app-doc/doxygen"
+
+KEYWORDS="~amd64 ~x86"
+
+PATCHES=(
+	"${FILESDIR}/${P}-20170213.patch"
+	"${FILESDIR}/${P}-allow-custom-build-type.patch"
+)
+
+mycmakeargs=( -DCMAKE_INSTALL_DOCDIR="share/doc/${PF}/html" )
```

Comment:
this will fail in EAPI 7. You should explicitly install html files to `html/` of `CMAKE_INSTALL_DOCDIR`, not change the canonical docdir such that it works.